lib/mars_rover.rb

```diff
@@ -0,0 +1,22 @@
+class MarsRover
+  attr_reader :direction
+  def initialize(x:, y:, direction:)
+    @x = x
+    @y = y
```

Comment:
x and y clump tother to form coordinates and you have a method called `coordinates`. That may be a signal of a new element (object) in your domain model.